index.html

```diff
@@ -1,44 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Neuron</title>
-    </head>
 
-    <body>
-        <!-- Header start -->
-        <!-- Header end -->
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="apple-touch-icon" sizes="180x180" href="./favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="./favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="./favicon/favicon-16x16.png">
+    <link rel="manifest" href="./favicon/site.webmanifest">
+    <link rel="mask-icon" href="./favicon/safari-pinned-tab.svg" color="#5bbad5">
+    <meta name="msapplication-TileColor" content="#603cba">
+    <meta name="theme-color" content="#ffffff">
 
-        <!-- Hero start -->
-        <!-- Hero end -->
+    <title>Neuron</title>
+</head>
 
-        <!-- About Us start -->
-        <!-- About Us end -->
+<body>
+    <!-- Header start -->
+    <!-- Header end -->
 
-        <!-- Stats start -->
-        <!-- Stats end -->
+    <!-- Hero start -->
+    <!-- Hero end -->
 
-        <!-- Services start -->
-        <!-- Services end -->
+    <!-- About Us start -->
+    <!-- About Us end -->
 
-        <!-- Projects start -->
-        <!-- Projects end -->
+    <!-- Stats start -->
+    <!-- Stats end -->
 
-        <!-- Subscribe start -->
-        <!-- Subscribe end -->
+    <!-- Services start -->
+    <!-- Services end -->
 
-        <!-- How We Work start -->
-        <!-- How We Work end -->
+    <!-- Projects start -->
+    <!-- Projects end -->
 
-        <!-- Blog Posts start -->
-        <!-- Blog Posts end -->
+    <!-- Subscribe start -->
+    <!-- Subscribe end -->
 
-        <!-- Contacts start -->
-        <!-- Contacts end -->
+    <!-- How We Work start -->
+    <!-- How We Work end -->
 
-        <!-- Footer start -->
-        <!-- Footer end -->
-    </body>
-</html>
+    <!-- Blog Posts start -->
+    <!-- Blog Posts end -->
+
+    <!-- Contacts start -->
+    <!-- Contacts end -->
+
+    <!-- Footer start -->
+    <!-- Footer end -->
+</body>
```

Comment:
čia iš esmės matau keičia visą formatavimą, bet dėl šito tai viskas gerai, po to kitą pull requestą supushinsim kur irgi keičia tai žėsim kokie konfliktai bus :D